### ISSUE TITLE
Google account password  stored as plain text?

### ISSUE THREAD

**Issue by aalexandrov** [nvbn/rhythmbox-gmusic#33]
If I am not mistaken, the following line from the `__init__.py` script stores the user's Google account password as plain text into the GGonf system.

```
settings.set_string(PASSWORD_KEY, dialog.password_input.get_text())
```

This is should be avoided at all cost (and I hope I don't have to explain why)!!!

Unfortunately, the plugin does not work properly on my machine (Ubuntu 12.04 using the pip install), so I probably won't be able to suggest a fix for that, but I suggest to use the Google OAuth 2.0 API and store the obtained access tokens instead.

**Comment by Nxt3** [comment]
Is there any way to delete our currently saved password?

**Comment by p0lar-bear** [comment]
Yes - passwords now are stored in the Gnome Keyring. Use a program like Seahorse to manage it.
